recipe/meta.yaml

```diff
@@ -39,7 +40,6 @@ requirements:
     - six >=1.7
     - munch
     - shapely
```

Comment:
Could you add a comment for `shapely`:   `# only option calc we include by default`? It's from the conda-forge recipe https://github.com/conda-forge/fiona-feedstock/blob/11831304485df0d40f9e7fbd0b46543cf9db89a1/recipe/meta.yaml#L46